psse/psse-converter/src/test/java/com/powsybl/psse/converter/PsseImporterIllinoisTest.java

```diff
@@ -43,7 +43,7 @@ void testLiteratureBasedIeee30() {
 
     @Test
     void testLiteratureBasedIeee39() {
-        testInvalid("/illinois/literature-based", "IEEE 39 bus.RAW", "Parsing error");
+        testValid("/illinois/literature-based", "IEEE 39 bus.RAW");
     }
 
     @Test
```

Comment:
Maybe we could delete the below associated fixed test and the fixed case since the original case is now imported correctly